lidar_mapping:
  
  method_type: 0  
  tf_btol: 
    [ 1, 0, 0, 0,
      0, 1, 0, 0,
      0, 0, 1, 0,
      0, 0, 0, 1 ]
  voxel_leaf_size: 1.0
  ndt_res: 2.0
  step_size: 0.1
  trans_eps: 0.01
  max_iter: 30
  min_scan_range: 5.0
  max_scan_range: 50.0
  min_add_scan_shift: 1.0

  cloud_input: "/points_no_ground"
  cloud_output: "/final_cloud"
  pose_out: "/pose_out"
  